data/qualification-level.tsv

```diff
@@ -1,17 +1,17 @@
-qualification-level	name	european-qualification-framework-level	start-date	end-date
-EntryLevel	Entry Level			2016-07-07
-Entry1	Entry 1			
-Entry2	Entry 2			
-Entry3	Entry 3	Level 1		
-Entry1_2	Entry 1,2			
-Entry2_3	Entry 2,3			
-Entry1_2_3	Entry 1,2,3			
-Level1	Level 1	Level 2		
-Level1_2	Level 1/2	Level 2/3		
-Level2	Level 2	Level 3		
-Level3	Level 3	Level 4		
-Level4	Level 4	Level 5		
-Level5	Level 5	Level 5		
-Level6	Level 6	Level 6		
-Level7	Level 7	Level 7		
-Level8	Level 8	Level 8		
+qualification-level	name	qualification-sub-level	european-qualification-framework-level	start-date	end-date
+EntryLevel	Entry Level				2016-07-07
+Entry1	Entry Level	Entry Level 1			
+Entry2	Entry Level	Entry Level 2			
+Entry3	Entry Level	Entry Level 3	Level 1		
+Entry1_2	Entry Level	Entry Level 1,2			
+Entry2_3	Entry Level	Entry Level 2,3			
+Entry1_2_3	Entry Level	Entry Level 1,2,3			
+Level1	Level 1		Level 2		
+Level1_2	Level 1/2		Level 2/3		
```

Comment:
why use / here instead of ,

Reply:
The custodian says:
* qualifications at `Level 1/2` can be awarded at either Level 1 or Level 2
* qualifications at `Entry Level 1,2` are awarded for both levels at the same time, so if you have to complete Levels 1, 2 and 3, then there's a difference between qualifying at Level 2 only, and Level 1 and 2 together -- in the first scenario you still have to complete Level 1.  Seems strange but that's apparently the way it works.